service/dps/storm/common/src/main/java/eu/europeana/cloud/service/dps/storm/StormTaskTuple.java

```diff
@@ -125,6 +119,10 @@ public Revision getRevisionToBeApplied() {
         return revisionToBeApplied;
     }
 
+    public void setRevisionToBeApplied(Revision revision) {
```

Comment:
we need to be able to null it (make outputRevision null) after using it inside (RevisionWritter); so we don't pass unnecessary information to the next bolt (AddResultToDataSetBolt in this case) .